crowdin/languages.go

```diff
@@ -0,0 +1,105 @@
+package crowdin
+
+import (
+	"context"
+	"fmt"
+	"net/http"
+
+	"github.com/crowdin/crowdin-api-client-go/crowdin/model"
+)
+
+// LanguagesService handle communication with Languages methods.
```

Comment:
Yes, I meant that we should put here the description from the API documentation. Since the comments are going to be extracted to the docs, these comments should be oriented to the end-user and follow the original API docs structure and way of documenting